docs/pages/appendices.md

```diff
@@ -0,0 +1,303 @@
+	</tr>
+</table>
+
+
+<a id="B" />
+### Appendix B: Run JSON Fields
+<!-- TODO: Convert this to explanations rather than checkmarks, 
+esp bc they might vary in meaning b/w PromION and MinION -->
+<table>
+	<tr>
+		<th>Field Name</th>
+		<th>PacBio</th>
+		<th>Illumina</th>
+		<th>Oxford Nanopore</th>
+	</tr>
+	<tr>
+		<td>platform</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>runAlias</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>sequencerFolderPath</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>sequencerName</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>sequencerPosition</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>containerSerialNumber</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>containerModel</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>laneCount</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>healthType</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>startDate</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>completedDate</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>pairedEndRun</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>software</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>metrics</td>
+		<td>✓</td>
+		<td>✓</td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>poolNames</td>
+		<td>✓</td>
+		<td></td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>runBasesMask</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>bclCount</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>callCycle</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>chemistry</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>imgCycle</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>indexLengths</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>numCycles</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>numReads</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>poolNames</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>readLength</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>scoreCycle</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>workflowType</td>
+		<td></td>
+		<td>✓</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>runType</td>
+		<td></td>
+		<td></td>
+		<td>✓</td>
+	</tr>
+	<tr>
+		<td>protocolVersion</td>
+		<td></td>
+		<td></td>
+		<td>✓</td>
+	</tr>
+</table>
+
+
+
+
+<a id="C" />
+### Appendix C: Sequencers Supported by Run Scanner
+This is a list of sequencers for which Run Scanner is capable of processing 
+output. 
+**Bold** items are verified to work with Run Scanner. 
+_Italic_ items are under development. Other items should work through having 
+equivalent output, however they have not been tested. 
```

Comment:
What do you think of rephrasing the `Other items [...]` sentence to something like `Other items are expected to work so long as the sequencer's output is in the same format as a sequencer whose name is in bold, but they have not been tested.` 
Either way, could it be put on a different line from the 'under development' line?

Reply:
Updated, please let me know what you think

Reply:
Looks good to me!